Bug: Skycoin-cli transaction command panics if zero args passes

Following some testing of the `skycoin-cli` `transaction` command it appears that the application will panic if you don't pass any arguments. Examination of the code indicates it is expecting one argument to always be passed.

Example:
```bash
$ skycoin-cli transaction     
```

Result:
```bash            
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/skycoin/skycoin/src/cli.transactionCmd.func1(0xc0002c6a00, 0x1b51680, 0x0, 0x0, 0x0, 0x0)
        /Users/BigOokie/Documents/_Dev/go/src/github.com/skycoin/skycoin/src/cli/transaction.go:28 +0x1b0
github.com/skycoin/skycoin/vendor/github.com/spf13/cobra.(*Command).execute(0xc0002c6a00, 0x1b51680, 0x0, 0x0, 0xc0002c6a00, 0x1b51680)
        /Users/BigOokie/Documents/_Dev/go/src/github.com/skycoin/skycoin/vendor/github.com/spf13/cobra/command.go:762 +0x465
github.com/skycoin/skycoin/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc000257180, 0x20, 0xc00027c3d0, 0xf)
        /Users/BigOokie/Documents/_Dev/go/src/github.com/skycoin/skycoin/vendor/github.com/spf13/cobra/command.go:852 +0x2ec
github.com/skycoin/skycoin/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /Users/BigOokie/Documents/_Dev/go/src/github.com/skycoin/skycoin/vendor/github.com/spf13/cobra/command.go:800
main.main()
        /Users/BigOokie/Documents/_Dev/go/src/github.com/skycoin/skycoin/cmd/cli/cli.go:26 +0x18c
```

The issue appears to be with the `Args` validation (below):
https://github.com/skycoin/skycoin/blob/be14cd713717f6c359fecd796240b08b370c38c3/src/cli/transaction.go#L26

Which then causes the following line of code to fail:
https://github.com/skycoin/skycoin/blob/be14cd713717f6c359fecd796240b08b370c38c3/src/cli/transaction.go#L28

Instead of using `cobra.MaximumNArgs(1)` should use `cobra.ExactArgs(1)`

Testing the implementation of the `Transaction` API, it appears it requires a transaction ID to be specified otherwise it will respond with the following:
```
Error: 400 Bad Request - txid is empty
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Return v2-style error for disabled endpoints
 - #2172 Fix electron build failure for linux system
 - Don't send messages that exceed the configured 256kB limit, which caused peers to disconnect from the sender
+- #2348 Fix panic in `skycoin-cli` `transaction` command if no (zero) arguments are passed. Exactly one argument is expected.
 
 ### Changed
 

--- a/src/cli/transaction.go
+++ b/src/cli/transaction.go
@@ -97,7 +97,7 @@ func getAddressTransactionsCmd(c *cobra.Command, args []string) error {
 		}
 	}
 
-	// If one or more addresses have beeb provided, request their transactions - otherwise report an error
+	// If one or more addresses have been provided, request their transactions - otherwise report an error
 	if len(addrs) > 0 {
 		outputs, err := apiClient.TransactionsVerbose(addrs)
 		if err != nil {

--- a/src/cli/transaction.go
+++ b/src/cli/transaction.go
@@ -23,7 +23,7 @@ func transactionCmd() *cobra.Command {
 		Use:                   "transaction [transaction id]",
 		DisableFlagsInUseLine: true,
 		SilenceUsage:          true,
-		Args:                  cobra.MaximumNArgs(1),
+		Args:                  cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
 			txid := args[0]
 			if txid == "" {

